lms/static/sass/shared-v2/_components.scss

```diff
@@ -30,7 +30,7 @@
             border-style: solid;
             border-width: 0 0 4px 0;
             border-bottom-color: transparent;
-            color: $lms-label-color;
+            color: rgb(94, 94, 94);
```

Comment:
To avoid the conflict, maybe you should just define a new variable in _variable.scss file.